go.sum

```diff
@@ -24,7 +24,6 @@ github.com/bytedance/sonic/loader v0.1.0/go.mod h1:UmRT+IRTGKz/DAkzcEGzyVqQFJ7H9
 github.com/bytedance/sonic/loader v0.1.1 h1:c+e5Pt1k/cy5wMveRDyk2X4B9hF4g7an8N3zCYjJFNM=
 github.com/bytedance/sonic/loader v0.1.1/go.mod h1:ncP89zfokxS5LZrJxl5z0UJcsk4M4yY2JpfqGeCtNLU=
 github.com/census-instrumentation/opencensus-proto v0.2.1/go.mod h1:f6KPmirojxKA12rnyqOA5BBL4O983OfeGPqjHWSTneU=
-github.com/choleraehyq/pid v0.0.18 h1:O7LLxPoOyt3YtonlCC8BmNrF9P6Hc8B509UOqlPSVhw=
 github.com/choleraehyq/pid v0.0.18/go.mod h1:uhzeFgxJZWQsZulelVQZwdASxQ9TIPZYL4TPkQMtL/U=
```

Comment:
It seems that we also need to remove this dependency.

Reply:
@liu-song it cannot rm and gen by go mod tidy. go mod graph shows it's be used by:

```
github.com/cloudwego/kitex@v0.9.3-rc github.com/choleraehyq/pid@v0.0.18
github.com/cloudwego/kitex@v0.9.3-dep3 github.com/choleraehyq/pid@v0.0.18
```

just leave it here first

Reply:
OK